util/serial/src/lib.rs

```diff
@@ -123,6 +133,64 @@ mod json_u64 {
 #[cfg(feature = "serde_with")]
 pub use json_u64::JsonU64;
 
+#[cfg(feature = "serde_with")]
+mod json_u128 {
```

Comment:
Optional/Equivalent: Merge the two mods into a `mod json_uint` and then `pub use json_unit::{JsonU64, JsonU128}`